docs/source/intro/overview.rst

```diff
@@ -212,6 +212,25 @@ Agent can run both from container and from a local machine.
 
         python3 -m core.run -ch telegram
 
+    or via HTTP api client:
```

Comment:
1. Install dev_requirements.txt
2. cd to `docs/`
3. build with `make html`
4. fix errors